<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Reminder</class>
 <widget class="QWidget" name="Reminder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>398</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Reminder{
	background-image: url(:/icons/image3.jpg);
}
QPlainTextEdit{
	background:rgba(0,0,0,0);
	border:1px none white;
	border-bottom:1px solid white;
	color:white;
}
QGroupBox{
	border:1px none white;
}
QLabel{
	color:white;
}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体_GB2312</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>你想在</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Date">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体_GB2312</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>（日期）</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>60</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体_GB2312</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>做什么？</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体_GB2312</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>标题</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体_GB2312</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>内容</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>440</y>
     <width>321</width>
     <height>81</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>5</horstretch>
     <verstretch>5</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background:rgb(170, 255, 255);
}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="pushButton_save">
      <property name="text">
       <string>保存</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="pushButton_clear">
      <property name="text">
       <string>清除</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="pushButton_close">
      <property name="text">
       <string>关闭</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>411</width>
     <height>551</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	background-image: url(:/icons/image3.jpg);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_Title">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>130</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正小标宋简体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_Content">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>190</y>
     <width>231</width>
     <height>231</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体_GB2312</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPlainTextEdit{
	color:black;
}</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="timeEdit_selectTime">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>118</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTimeEdit
{
    min-height:24px;
    max-height:24px;
    border-radius:2px;
    border:1px solid #cccccc;
	background:rgb(170, 255, 255);
}

QTimeEdit:hover,
QTimeEdit:focus
{
    border-radius:2px;
    border:1px solid #0f6dbe;
}

/*QTimeEdit::down-button
{
    border:none;
    width:30px;
    background-color:white;
}


.QTimeEdit::up-button
{
    border:none;
    width:30px;
    background-color:white;
}*/

</string>
   </property>
   <property name="displayFormat">
    <string>HH:mm</string>
   </property>
  </widget>
  <zorder>label_2</zorder>
  <zorder>label</zorder>
  <zorder>label_Date</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>groupBox</zorder>
  <zorder>plainTextEdit_Title</zorder>
  <zorder>plainTextEdit_Content</zorder>
  <zorder>timeEdit_selectTime</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_close</sender>
   <signal>clicked()</signal>
   <receiver>Reminder</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>303</x>
     <y>470</y>
    </hint>
    <hint type="destinationlabel">
     <x>198</x>
     <y>286</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_clear</sender>
   <signal>clicked()</signal>
   <receiver>plainTextEdit_Title</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>480</y>
    </hint>
    <hint type="destinationlabel">
     <x>235</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_clear</sender>
   <signal>clicked()</signal>
   <receiver>plainTextEdit_Content</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>480</y>
    </hint>
    <hint type="destinationlabel">
     <x>235</x>
     <y>305</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
